### PR TITLE
Equivalent values for all tokens (ETH + ERC20s)

### LINKED ISSUE #90: Balance Component
- [x] ETH Balance
- [x] Token Balances
- [x] User's address + addressicon
- [x] Donation Blurb on Send Page
- [x] Equivalent values in USD CHF BTC EUR GBP
- [x] Equivalent values for Tokens
- [ ] Equivalent values for all balances (ether + tokens)

Github Ref
1. Better ethplorer integration? kvhnuke/etherwallet: Issue #431
2. GBP kvhnuke/etherwallet: Issue #440
3. Equiv of tokens kvhnuke/etherwallet: Issue #435

### PR DESCRIPTION
**Pull request by wbobeirne**
This is a continuation of #366 and finally closes #90. This adds an option (and makes it the default) to show equivalent values of your ETH balance as well as your tokens.

To reduce roundtrips, this calculates token values by going from token -> eth -> usd / gbp etc., instead of using the API to go directly from token to other. This may result in some _very_ minor differences, but it should be pretty small. Perhaps we should add a note here saying these are rough values and should not be taken as facts.

Here's what it looks like:

screen shot 2017-11-17 at 1 13 27 pm

### CODE REVIEW

File: common/components/BalanceSidebar/EquivalentValues.tsx [hunk 1]
```diff
@@ -63,23 +68,19 @@ export default class EquivalentValues extends React.Component<Props, CmpState> {
...
-              {balance.isPending ? (
-                <Spinner />
-              ) : (
-                <UnitDisplay
-                  unit={'ether'}
-                  value={balance ? balance.muln(rates[currency][key]) : null}
-                  displayShortBalance={2}
-                />
-              )}
+              <UnitDisplay
```
Comment (dternyak) [inline common/components/BalanceSidebar/EquivalentValues.tsx]: Why get rid of the spinner?

Comment (wbobeirne) [reply common/components/BalanceSidebar/EquivalentValues.tsx]: The spinner should never happen; It was replaced with the one large spinner on line 83.

File: common/actions/rates/actionPayloads.ts [hunk 2]
```diff
@@ -1,31 +1,52 @@
...
+const CCRates = (symbols: string[]) => {
+  const tsyms = rateSymbols.concat(symbols).join(',');
+  return `${CCApi}/data/price?fsym=ETH&tsyms=${tsyms}`;
 };
 
 export interface CCResponse {
-  symbol: string;
-  rates: {
-    BTC: number;
+  [symbol: string]: {
```
Comment (skubakdj) [inline common/actions/rates/actionPayloads.ts]: This should break the rates reducer test recently merged into `develop`.

File: common/components/BalanceSidebar/EquivalentValues.tsx [hunk 3]
```diff
@@ -63,23 +68,19 @@ export default class EquivalentValues extends React.Component<Props, CmpState> {
...
+                unit={'ether'}
+                value={values[key]}
+                displayShortBalance={3}
```
Comment (skubakdj) [inline common/components/BalanceSidebar/EquivalentValues.tsx]: Should we up this to `5`? Low-value accounts (~$1) currently get rounded out to `0 BTC` in the equiv rates display.

File: common/actions/rates/actionPayloads.ts (hunk 2 above)
Comment (wbobeirne) [reply common/actions/rates/actionPayloads.ts]: Good catch, fixed!

File: common/components/BalanceSidebar/EquivalentValues.tsx (hunk 3 above)
Comment (wbobeirne) [reply common/components/BalanceSidebar/EquivalentValues.tsx]: I was talking to henry about this, I think we need to have UnitDisplay show `< 0.001` when the value is below the threshold, but above 0.

File: common/components/BalanceSidebar/EquivalentValues.tsx (hunk 1 above)
Comment (wbobeirne) [reply common/components/BalanceSidebar/EquivalentValues.tsx]: Un-redacted, this is fixed.

**Pull request merged**